### PR TITLE
[FIX] *: test pylint `%s`

### PR DESCRIPTION
**Pull request by ngochung207**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/loyalty/i18n/fa.po [hunk 1]
```diff
@@ -936,7 +936,7 @@ msgstr "مشتریان"
 #: model_terms:ir.ui.view,arch_db:loyalty.loyalty_report
 msgid "DEMO_CODE"
 msgstr ""
-"متن ارائه شده حاوی عباراتی به زبان انگلیسی است که به یک ماژول خاص از Odoo، یعنی ماژول وفاداری (/home/esmaeil/odoo17/odoo/odoo/addons/loyalty) اشاره دارد. به همین دلیل، لازم است برای ترجمه به زبان فارسی، دقت ویژه‌ای به کار برده شود تا اطمینان حاصل شود که مفاهیم منتقل شده صحیح و دقیق هستند و با شرایط و اصطلاحات مرسوم در Odoo تطابق دارند. هنگام ترجمه، ساختار HTML نیز باید حفظ شود و متن‌های بین تگ‌ها به درستی ترجمه شوند. پلاسبولدرهای موجود مانند %، %s یا %d نباید تغییر کنند و همانطور که هستند در جاهای خود باقی بمانند. \n"
+"متن ارائه شده حاوی عباراتی به زبان انگلیسی است که به یک ماژول خاص از Odoo، یعنی ماژول وفاداری (/home/esmaeil/odoo17/odoo/odoo/addons/loyalty) اشاره دارد. به همین دلیل، لازم است برای ترجمه به زبان فارسی، دقت ویژه‌ای به کار برده شود تا اطمینان حاصل شود که مفاهیم منتقل شده صحیح و دقیق هستند و با شرایط و اصطلاحات مرسوم در Odoo تطابق دارند. هنگام ترجمه، ساختار HTML نیز باید حفظ شود و متن‌های بین تگ‌ها به درستی ترجمه شوند. پلاسبولدرهای موجود مانند ،  یا نباید تغییر کنند و همانطور که هستند در جاهای خود باقی بمانند. \n"
 "\n"
```
Comment (royle-vietnam) [inline addons/loyalty/i18n/fa.po]: @ngochung207 xóa cả đi chứ anh
msgid "DEMO_CODE" chỉ có thế này thôi mà
```suggestion
```

File: odoo/addons/base/i18n/et.po [hunk 2]
```diff
@@ -12588,6 +12588,7 @@ msgid ""
 "An SSL exception occurred. Check connection security type.\n"
 " %s"
 msgstr "Ilmnes SSL erand . Kontrolli ühenduse turvalisuse tüüpi."
```
Comment (royle-vietnam) [inline odoo/addons/base/i18n/et.po:12590]: @ngochung207 cho nó xuống dòng nữa
```suggestion
msgstr "Ilmnes SSL erand . Kontrolli ühenduse turvalisuse tüüpi.\n"
```

File: addons/hr_holidays/i18n/fa.po [hunk 3]
```diff
@@ -176,7 +176,7 @@ msgstr "%s (کپی)"
 #: code:addons/hr_holidays/models/hr_leave_allocation.py:0
 #, python-format
 msgid "%s (from %s to %s)"
-msgstr "٪s (از ٪s تا ٪s)"
+msgstr "%s (از ٪s تا %s)"
```
Comment (royle-vietnam) [inline addons/hr_holidays/i18n/fa.po]: @ngochung207 vẫn sót chỗ này anh

**Review by royle-vietnam — APPROVED** [review]
LGTM

**Comment by ngochung207** [comment]
@viinbot rebuild

**Comment by ngochung207** [comment]
[Build Details _ Viindoo Runbot_v17.pdf](https://github.com/user-attachments/files/17912579/Build.Details._.Viindoo.Runbot_v17.pdf)

**Pull request merged**